x-pack/plugins/features/server/feature_privilege_iterator/feature_privilege_iterator.ts

```diff
@@ -151,6 +151,14 @@ function mergeWithSubFeatures(
         mergedConfig.cases?.settings ?? [],
         subFeaturePrivilege.cases?.settings ?? []
       ),
+      createComment: mergeArrays(
```

Comment:
nit: would you mind including these in the relevant unit test in `x-pack/plugins/features/server/feature_privilege_iterator/feature_privilege_iterator.test.ts`

Reply:
ya will update, although that seems like something that typescript should catch

Reply:
updated